packages/core/template/default/gitignore

```diff
@@ -20,4 +20,4 @@ _site/
 # IDE configs
 .vscode/
 .idea/*
-*.iml
\ No newline at end of file
+*.iml
```

Comment:
unnecessary change

Reply:
I am not sure how to fix this problem because the changes were made by WebStorm (due to missing EoF line in the original code), I tried using GitHub to remove the last line but nothing happened https://github.com/MarkBind/markbind/pull/2469/commits/d62c7ac70ba76651108d74e85f50028ef83c10dc